Extended_Euclidean_Algorithm/Extended_Euclidean_Algorithm.ts

```diff
@@ -0,0 +1,32 @@
+export {}
+
+// find greatest common divisor
+function gcd(x:number , y:number) : number
+{
+    let rem : number;
+    while ((x % y) > 0)
+    {
+        rem = x % y;
+        x = y;
+        y = rem;
+    }
+
+    return y;
+}
+
+let x : number = 12;
```

Comment:
Take input dynamically.

Reply:
@atarax665 Actually in typescript we just cannot take it that way some changes need to be done in html file to do that because it basically deals with the browser window so in order to take different inputs we can directly change the values

Reply:
@shivanidalmia1203 Okay, got you